lps-client-backend/src/main/kotlin/no/nav/helsearbeidsgiver/plugins/Routing.kt

```diff
@@ -36,6 +39,46 @@ fun Application.configureRouting() {
     }
 }
 
+private fun Routing.registrerNyBedrift() {
+    post("/registrer-ny-bedrift") {
+        val parametre = call.receiveParameters()
+        val kundeOrgnr =
+            parametre["kundeOrgnr"] ?: return@post call.respond(
+                HttpStatusCode.BadRequest,
+                "Mangler 'kundeOrgnr' parameter",
+            )
+
+        logger().info("Prøver å registrere bedriften med orgnr: $kundeOrgnr som ny kunde.")
+        try {
+            val maskinportenClientConfig =
```

Comment:
Det er kanskje ikke så farlig siden det er en dev app. 
Men synes det er bedre å instansiere maskinporten klienten en gang.
Og sende den videre der den trengs.
Slik det er nå vil vel det instansieres en maskinporten klient for hvert api kall?

Reply:
Ja, enig. Skal se på det! 🫡